soh/src/code/z_actor.c

```diff
@@ -3142,6 +3143,40 @@ int gMapLoading = 0;
 
 Actor* Actor_Spawn(ActorContext* actorCtx, GlobalContext* globalCtx, s16 actorId, f32 posX, f32 posY, f32 posZ,
                    s16 rotX, s16 rotY, s16 rotZ, s16 params) {
+
+    // Hack to remove bats and skulltulas that spawn in graveyard because of removing object dependency.
+    if ((actorId == ACTOR_EN_FIREFLY || (actorId == ACTOR_EN_SW && params == 0)) && globalCtx->sceneNum == 83) {
```

Comment:
i figure it'll be good to use the scene enum here (just wish this entry had a better name than `SPOT02`)

```suggestion
    if ((actorId == ACTOR_EN_FIREFLY || (actorId == ACTOR_EN_SW && params == 0)) && globalCtx->sceneNum == SCENE_SPOT02) {
```

Reply:
This is just a temporary hack for testing the PR. It's a side-effect of removing the object dependency, and I believe when it was discussed on discord, the idea was to either patch these at OTR generation or when the OTR is loaded. So this whole block of code is hopefully not going to be required once we have proper fixes for these.

I'll change the scene num to `SCENE_SPOT02` until then anyway, just figured I'd clarify.